wordfreq_builder/wordfreq_builder/tokenizers.py

```diff
@@ -95,19 +116,17 @@ def tokenize_by_language(in_filename, out_prefix, tokenizer):
     Produces output files that are separated by language, with spaces
     between the tokens.
     """
-    out_files = {}
+    out_files = {
+        language: open('%s.%s.txt' % (out_prefix, language), 'w', encoding='utf-8')
+        for language in KEEP_THESE_LANGUAGES
+    }
     with open(in_filename, encoding='utf-8') as in_file:
         for line in in_file:
             text = line.split('\t')[-1].strip()
             language, tokens = tokenizer(text)
-            if language != 'un':
+            if language in KEEP_THESE_LANGUAGES:
```

Comment:
(really tiny nitpick that you can feel free to toss: this is probably more direct and change-proof as `if language in out_files:`)